test/pdo_sqlsrv/PDO_ConnPool_Unix.phpt

```diff
@@ -4,32 +4,60 @@ PDO Connection Pooling Test on Unix
 This test assumes odbcinst.ini has not been modified. 
 This test also requires root privileges to modify odbcinst.ini file on Linux.
 --SKIPIF--
-<?php if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') die("Skipped: Test for Linux and Mac");
+<?php if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') die("Skipped: Test for Linux and Mac"); ?>
 --FILE--
 <?php
+// On Bamboo we must use sudo to fiddle with odbcinst.ini
+// On travis-ci we can't use sudo
+$sudo = '';
+$user = posix_getpwuid(posix_geteuid());
+if (strtolower($user['name']) == 'bamboo')
+{
+    $sudo = 'sudo ';
+}
+
```

Comment:
Can we generalize this by checking if it's `root` instead? So if `$user['name'] == 'root'` then do not use `sudo`
Also please modify the comment to reflect this change.